pkgs/development/tools/aws-sam-cli/default.nix

```diff
@@ -37,14 +40,6 @@ python3.pkgs.buildPythonApplication rec {
   '';
 
   patches = [
-    # Click 8.1 removed `get_terminal_size`, recommending
-    # `shutil.get_terminal_size` instead.
-    # (https://github.com/pallets/click/pull/2130)
-    ./support-click-8-1.patch
-    # Werkzeug >= 2.1.0 breaks the `sam local start-lambda` command because
-    # aws-sam-cli uses a "WERKZEUG_RUN_MAIN" hack to suppress flask output.
-    # (https://github.com/cs01/gdbgui/issues/425)
-    ./use_forward_compatible_log_silencing.patch
   ];
```

Comment:
If these batches aren't used, can we remove the patch files?

Reply:
This can not be patched, so I erased it, but I need to check it carefully.